src/actions/DeepLinkingActions.tsx

```diff
@@ -226,6 +227,9 @@ async function handleLink(navigation: NavigationBase, dispatch: Dispatch, state:
       const parseWallets = async (): Promise<void> => {
         // Try to parse with all wallets
         for (const wallet of Object.values(currencyWallets)) {
+          // Ignore disabled wallets:
+          if (SPECIAL_CURRENCY_INFO[wallet.currencyInfo.pluginId].keysOnlyMode) return
```

Comment:
Since keysOnlyMode can be undefined, we should do `const { keysOnlyMode = false } = SPECIAL_CURRENCY_INFO`, and then you can do if (keysOnlyMode) on the next line. That makes the default value explicit.